web-components/src/components/tabs/Tabs.ts

```diff
@@ -860,6 +861,7 @@ export namespace Tabs {
           } else if (tab && !tab.disabled) {
             const idx = this.tabsIdxHash[this.getNormalizedTabId(tab.id)];
             if (idx !== -1) {
+              event.preventDefault();
               this.updateSelectedTab(idx);
             }
```

Comment:
fix issue where space to active also scrolled scroll view as prevent default wan't called